README.md

```diff
@@ -19,6 +19,12 @@ See `pillar.example` to configure clean up behavior.
 
 Install `docker-compose` package.
 
+### docker.dnsmasq
+
+Setup `dnsmasq` to listen as a DNS server on the docker gateway `172.17.0.1`. This is useful when dns servers configured on the host are ipv6 only
```

Comment:
This is also useful when you want to specify a DNS for all docker instances, as it sometimes defaults to `8.8.8.8`.

Reply:
Not sure about this, as long as the DNS server(s) specified on the host are reachable for docker, those would be used. `8.8.8.8` is the fallback if they are unreachable like in the case of ipv6 nameservers. To specify an ipv4 name server for all instances, it just needs to be added in `daemon.json` or on the host i.e `/etc/resolv.conf`

Reply:
I've seen multiple container images using `8.8.8.8` instead of the ones configured on the host. Especially with things like `systemd-resolved`, it might sometimes be hard for `dockerd` to automatically infer the right host nameservers.

Example:


```bash
$ docker run -ti alpine cat /etc/resolv.conf
nameserver 8.8.8.8
nameserver 8.8.4.4
```

Reply:
But this might be due to ipv6 being used on the host.

Reply:
I think we should just document this in the README then that's ok.

Reply:
I guess a reference to the dns docs should be good enough?

- https://docs.docker.com/config/containers/container-networking/#dns-services

Reply:
I was thinking about something like below, but a link to the docs is a good idea, too!

> ... lack of NAT, as Docker defaults to Google's DNS servers (`8.8.8.8` and `8.8.4.4`) for its containers when the nameserver configuration of the host only specifies ipv6 addresses.